Makefile

```diff
@@ -42,7 +42,7 @@ pkg-show-update:
 
 pkg-freeze-setup:
 	python3 tools/dev/poetrypkg.py gen-frozensetup -p .
-	for pkg in $(PKG_SET); do python3 tools/dev/poetrypkg.py gen-frozensetup -p $$pkg -x sendgrid -r starkbank-ecdsa; done
+	for pkg in $(PKG_SET); do python3 tools/dev/poetrypkg.py gen-frozensetup -p $$pkg -x pywin32; done
```

Comment:
I was giving this a spin locally and got a traceback here:

```
ython3 tools/dev/poetrypkg.py gen-frozensetup -p .                                                                                                  
for pkg in tools/c7n_gcp tools/c7n_azure tools/c7n_kube tools/c7n_openstack tools/c7n_mailer tools/c7n_logexporter tools/c7n_policystream tools/c7n_t
railcreator tools/c7n_org tools/c7n_sphinxext tools/c7n_terraform tools/c7n_awscc; do python3 tools/dev/poetrypkg.py gen-frozensetup -p $pkg -x pywin
32; done                                                                  
Traceback (most recent call last):                                                                                                                   
  File "/home/aj/code/cloud-custodian/release/tools/dev/poetrypkg.py", line 220, in <module>                                                         
    cli()                                                                                                                                            
  File "/home/aj/.pyenv/versions/3.10.0/envs/c7n-release/lib/python3.10/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)                                                                                                                
  File "/home/aj/.pyenv/versions/3.10.0/envs/c7n-release/lib/python3.10/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "/home/aj/.pyenv/versions/3.10.0/envs/c7n-release/lib/python3.10/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/aj/.pyenv/versions/3.10.0/envs/c7n-release/lib/python3.10/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/aj/.pyenv/versions/3.10.0/envs/c7n-release/lib/python3.10/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "/home/aj/code/cloud-custodian/release/tools/dev/poetrypkg.py", line 158, in gen_frozensetup
    setup_content = builder.build_setup()
  File "/home/aj/.poetry/lib/poetry/_vendor/py3.10/poetry/core/masonry/builders/sdist.py", line 176, in build_setup
    dependencies, extras = self.convert_dependencies(
  File "/home/aj/code/cloud-custodian/release/tools/dev/poetrypkg.py", line 153, in convert_dependencies
    reqs, default = locked_deps(package, poetry, exclude, remove)
  File "/home/aj/code/cloud-custodian/release/tools/dev/poetrypkg.py", line 206, in locked_deps
    reqs.append(project_deps[p.name].to_pep_508())
KeyError: 'pywin32'
```

Should this bit be `-r pywin32` instead of `-x pywin32`?

Reply:
indeed it should